src/views/choose-registered-office-address.njk

```diff
@@ -36,19 +36,23 @@
               classes: "govuk-fieldset__legend--xl"
             }
           },
-          items: addressItems
+          items: addressItems,
+          attributes: {
+            required : true,
+            "radio-event-id": "address-radio-selected"
+          }
         }) }}
        
         <p>
           <a href="" class="govuk-link" data-event-id="registered-office-address-manual-link">{{i18n.chooseRegisteredOfficeAddressPage.addressLink }}</a>
         <p>
        
        {{ govukButton({
-               text: i18n.buttons.continue,
-               attributes: {
-               "id": "submit",
-               "onclick": "_paq.push(['trackEvent', ' Registerd Office address choice', 'selection', document.querySelector('input[name=parameter]:checked').nextElementSibling.innerHTML])"
-         }
+          text: i18n.buttons.continue,
+          attributes: {
+          "id": "submit",
```

Comment:
the attributes need indenting as they follow an open bracket {.
same as the attributes you have indented correctly on line 41